packages/cli/src/__tests__/commands/Studio.test.ts

```diff
@@ -27,89 +28,93 @@ function sendRequest(message: any): Promise<any> {
 }
 
 let studio: Studio
-describe('studio with alternative urls and prisma://', () => {
-  afterEach(() => {
-    // Back to original env vars
-    process.env = { ...originalEnv }
-  })
+// Prisma Studio ignores env vars for overriding engine paths, skipping test for now
```

Comment:
Would it help to store the engine path statically at build time (eg. inject the env var path contents at build time)?

Reply:
Additionally, making it static may allow not have have to touch any test in general, that would be similar to the approach I took here https://github.com/prisma/prisma/blob/551986bc5c4900eaf3d71f2725b327ccef7154e1/packages/fetch-engine/src/download.ts/#L62-L67, basically is `enginesOverride` exists, we will copy the engine into the `.cache/prisma/_local_` folder, making it transparent to any test that uses it.

Reply:
For quite a bunch of skipped tests that will indeed help to unskip them. There are few of the tests which test that reported engine version is the same as specified in `package.json` and those would have to have stay skipped.